includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -2964,14 +3006,32 @@ public static function get_source_key_label( $key, $validation_error ) {
 				}
 			case 'parent_name':
 				return __( 'Parent element', 'amp' );
-			case 'property_name':
+			case 'css_property_name':
 				return __( 'CSS property', 'amp' );
 			case 'text':
 				return __( 'Text content', 'amp' );
 			case 'type':
 				return __( 'Type', 'amp' );
 			case 'sources':
 				return __( 'Sources', 'amp' );
+			case 'property_name':
```

Comment:
Should this be renamed to `meta_property_name` then?

Reply:
Alternatively, the `$validation_error['code']` can be used to differentiate which property type is in view here. And that could avoid the need to use `css_property_name`. Thoughts?

Reply:
Personally, I prefer renaming it to `meta_property_name` to be more descriptive.

Reply:
Done via https://github.com/ampproject/amp-wp/pull/4137/commits/fa1299e503eeb853d8bb185d0b660e8eeec724bf.